Lots of changes -arthur

aeubanks opened the pull request:
Changed names of Activities to include "Activity"
Moved strings of all activities to their own xml file
Added an empty FriendsActivity class
Added ability to go back to previous activities
removed some finish() statements so that when going back they weren't destroyed
removed all menus (with settings)

Commit:
Changed names of Activities to include "Activity"
Moved strings of all activities to their own xml file
Added an empty FriendsActivity class
Added ability to go back to previous activities
removed some finish() statements so that when going back they weren't destroyed
removed all menus (with settings)

JoshVorick commented:
This pull request definitely does not have all your changes. Make sure you commited and pushed all the changes you made to the branch from which you're making the PR.

Commit:
Changed names of Activities to include "Activity"
Moved strings of all activities to their own xml file
Added an empty FriendsActivity class
Added ability to go back to previous activities
removed some finish() statements so that when going back they weren't destroyed
removed all menus (with settings)

aeubanks commented:
my very bad
